src/stdatamodels/stuserdict.py

```diff
@@ -0,0 +1,94 @@
+import _collections_abc
+# import sys as _sys
+
+# from itertools import chain as _chain
+# from itertools import repeat as _repeat
+# from itertools import starmap as _starmap
+# from keyword import iskeyword as _iskeyword
+# from operator import eq as _eq
+# from operator import itemgetter as _itemgetter
+# from reprlib import recursive_repr as _recursive_repr
+# from _weakref import proxy as _proxy
+
+# try:
+#     from _collections import deque
+# except ImportError:
+#     pass
+# else:
+#     _collections_abc.MutableSequence.register(deque)
+
+# try:
+#     from _collections import defaultdict
+# except ImportError:
+#     pass
+
+class STUserDict(_collections_abc.MutableMapping):
```

Comment:
This seems to be identical to the 3.8.0b1 version of [collections.UserDict](https://github.com/python/cpython/blob/v3.8.0b1/Lib/collections/__init__.py#L971), do we need that version for some reason?  Maybe we can just use the built-in UserDict?

Reply:
Which version to use is a good question. It took me a few minutes to recall why I couldn't use the built in, but then it hit me like a 2x4. The problem is the built-in has a data attribute, and that would conflict with our need for a data attribute (if we didn't use it, someone would), so the only difference in this version is that `.data` was changed to `._data`. Is there a better way to do it?

Reply:
Ohhhhhh ok!  That makes sense.  We may be able to get away with subclassing dict directly, since (at least in this code so far) we're not overriding any of the MutableMapping methods.